spec/acceptance/sensu_check_spec.rb

```diff
@@ -68,13 +68,6 @@
         expect(data['metadata']['namespace']).to eq('test')
       end
     end
-
```

Comment:
why would this change? it seems unrelated.

Reply:
Because it's fragile. We add/remove checks in other files and to constantly having to update this number based on unrelated tests seems like a bad idea. Also we are directly testing that the checks we created all exist so checking the count just adds fragility to the acceptance tests.

Reply:
This became an issue when I added checks to test the postgresql still worked and so an external test to this file was impacting this test which sort of violates the idea that tests are self contained.